Sources/SwiftParser/Parser.swift

```diff
@@ -250,11 +250,7 @@ public struct Parser {
   public var alternativeTokenChoices: [Int: [TokenSpec]] = [:]
 
   mutating func recordAlternativeTokenChoice(for lexeme: Lexer.Lexeme, choices: [TokenSpec]) {
-    guard let lexemeBaseAddress = lexeme.tokenText.baseAddress,
-      let offset = lexemes.offset(of: lexemeBaseAddress)
-    else {
-      return
-    }
+    let offset = lexemes.getOffsetToStart(lexeme)
```

Comment:
One nitpick since you’re changing this (I think I thought about this before but apparently never left a comment): Could you rename the function to `offsetToStart`. Swift generally doesn’t use the term `get` in function names.